services/Cargo.toml

```
@@ -97,13 +97,14 @@ zip = "0.6"
 nix = "0.26"
 
 [dev-dependencies]
+assert_cmd = "2.0"
 httptest = "0.15.2"
+hyper = "0.14.24"
+prost = "0.11.6"
```

```suggestion
hyper = "0.14"
prost = "0.11"
```

Could you omit the last digit? It doesn't matter anyway in this config.